src/Admin/AbstractAdmin.php

```diff
@@ -1735,6 +1761,14 @@ public function addChild(AdminInterface $child)
         $this->children[$child->getCode()] = $child;
 
         $child->setParent($this);
+
+        // TODO: remove $args and add $field parameter to this function on next Major
+
+        $args = \func_get_args();
+
+        if (isset($args[1])) {
+            $child->addParentAssociationMapping($this->getCode(), $args[1]);
```

Comment:
Well maybe we can throw a deprecation if second argument is not provided?

Reply:
sounds better, will fix it.

Reply:
done.